submissions/01samiksha/aboutme.md

```diff
@@ -0,0 +1,18 @@
+# About Me
+My name is Samiksha Gulati. I am a CSE student at UIET,Panjab University, India.
+
+# Experience
+None as such
+
+# Interests
+Programming
+
+# Contact Info
+Email: [01samiksha@gmail.com](mailto:01samiksha@gmail.com)
+
+Github: [01samiksha](https://github.com/01samiksha)
+
+# Emoji
+My favorite emoji is :thinking_face:
+
```

Comment:
Remove empty lines.

Reply:
Just the last extra line, or all the empty lines in b/w

Reply:
you can view the .md by clicking on the view button above, so depends on you. I found all lines removed also good but last lines surely to be removed.